NotebookExamples/csharp/Samples/DataFrame-Getting Started.ipynb

```diff
@@ -0,0 +1,387 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "Installing package Microsoft.Data.Analysis, version 0.1.0.....done!"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "Successfully added reference to package Microsoft.Data.Analysis, version 0.1.0"
```

Comment:
We've started clearing cell outputs before checking in the notebooks. We're open to other approaches. The reason is that a number of people have opened the notebooks and assumed cells have already executed, and that they can then jump in anywhere, at which point things blow up because prior cells were supposed to have been run.